pyproject.toml

```diff
@@ -37,10 +37,10 @@ aea = 'aea.cli:cli'
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
 base58 = ">=1.0.3,<3.0.0"
-jsonschema = "^4.0.0"
+jsonschema = ">=3.2.0,<5"
 packaging = "^21.0"
 semver = ">=2.9.1,<3.0.0"
-protobuf = ">=3.19.0,<=3.19.4"
+protobuf = ">=3.19.4,<4"
```

Comment:
Shouldn't we peg it to a specific version?

Reply:
no, it's better to stay it like this